src/textacy/constants.py

```diff
@@ -21,6 +21,51 @@
 OBJ_DEPS: set[str] = {"attr", "dobj", "dative", "oprd"}
 AUX_DEPS: set[str] = {"aux", "auxpass", "neg"}
 
+MIN_QUOTE_LENGTH: int=4
+
+QUOTATION_MARK_PAIRS = {
+    # """
+    # Ordinal points of the token.is_quote characters, matched up by start and end.
+
+    # Some of these pairs are from weirdly formatted newspaper uploads, so could be some noise!
+
+    # source:
+    # switch = "\"\'"
+    # start = "“‘```“‘«‹「『„‚"
+    # end = "”’’’’”’»›」』”’"
+    # """
+    (34, 34),       # " "
+    (39, 39),       # ' '
+    (96, 8217),     # ` ’
+    (171, 187),     # « »
+    (8216, 8217),   # ‘ ’
+    (8218, 8217),   # ‚ ’
+    (8220, 8221),   # “ ”
+    (8222, 8221),   # „ ”
+    (8249, 8250),   # ‹ ›
+    (12300, 12301), #「 」 
+    (12302, 12303), #『 』
+    (8220, 34),     # “ "
+    (8216, 34),     # ‘ "
+    (96, 34),       # ` "
+    (8216, 34),     # ‘ "
+    (171, 34),      # « "
+    (8249, 34),     # ‹ "
+    (12300, 34),    #「 "
+    (12302, 34),    #『 "
+    (8222, 34),     # „ "
+    (8218, 34),     # ‚ "
+    (34, 8221),      # " ”
+    (34, 8217), # " ’
+    (34, 10),
+    (39, 10),
+    (96, 10),
+    (171, 10),
+    (8216, 10),
+    (8218, 10),
+    (8249, 10)
+    }
+
```

Comment:
```suggestion
"""
Ordinal points of the token.is_quote characters, matched up by start and end.
Some of these pairs are from weirdly formatted newspaper uploads, so could be some noise!

source:
switch = "\"\'"
start = "“‘```“‘«‹「『„‚"
end = "”’’’’”’»›」』”’"
"""

```

Reply:
i don't think i understand the source/switch/start/end lines at the end. what are those saying?

Reply:
"start" characters can only start quotations, "end" characters can only end quotations, "switch" characters can do either. I felt like it was worth leaving them in there for reference.